src/app/Topology/Actions/utils.tsx

```diff
@@ -213,20 +212,22 @@ export const nodeActions: NodeAction[] = [
       services.api
         .graphql<GroupActionResponse>(
           `
-            query StopRecordingForGroup ($groupFilter: EnvironmentNodeFilterInput, $recordingFilter: ActiveRecordingFilterInput){
+            query StopRecordingForGroup ($groupFilter: DiscoveryNodeFilterInput, $recordingFilter: ActiveRecordingsFilterInput) {
               environmentNodes(filter: $groupFilter) {
                 name
                 descendantTargets {
                   name
-                  recordings {
+                  target {
+                    recordings {
                       active(filter: $recordingFilter) {
-                          data {
-                              doStop {
-                                name
-                                state
-                              }
+                        data {
+                          doStop {
+                            name
+                            state
```

Comment:
The data section is tabbed differently for all three active recording graphql requests in this file? Here, lines 179-181, and lines 269-272.